lib/Common/Memory/RecyclerPointers.h

```diff
@@ -375,6 +375,22 @@ class WriteBarrierPtr
         return result;
     }
 
+    WriteBarrierPtr& operator--()  // prefix --
+    {
+        --ptr;
+#ifdef RECYCLER_WRITE_BARRIER
+        RecyclerWriteBarrierManager::WriteBarrier(this);
+#endif
+        return *this;
+    }
+
+    WriteBarrierPtr operator--(int)  // postfix --
```

Comment:
that was because of quick sort code added here, but after I change the implementation it's not necessary. I kept it just for the completion - in case there's usage in the future. I can remove if you think it should not be here